### PR TITLE
done

### PR DESCRIPTION
**Pull request by 4uhaister**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "metarhia"
```
Comment (nosovk) [inline .eslintrc.json]: sonar lint забыт?

File: .husky/pre-commit [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm run format
+npm test
+npm run stylelint
+git add .
```
Comment (nosovk) [inline .husky/pre-commit]: не должно быть сайдеффектов в хуках

**Review by nosovk — CHANGES_REQUESTED** [review]

File: .idea/codeStyles/Project.xml [hunk 3]
```diff
@@ -0,0 +1,58 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (nosovk) [inline .idea/codeStyles/Project.xml:1]: это думаю тоже можно в ignore положить?

File: index.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
```
Comment (nosovk) [inline index.html:4]: забыли viewport

File: index.html [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    <title>WEB-1</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="wrapper">
+      <div id="b1" class="block"><h1>Сало це смачно!</h1></div>
+      <div id="b2" class="block">Картопляне пюре</div>
```
Comment (nosovk) [inline index.html]: id is evil, please avoid

File: index.html [hunk 6]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>WEB-1</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="wrapper">
```
Comment (nosovk) [inline index.html:11]: html semantic tags?

File: index.html [hunk 7]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>WEB-1</title>
+    <link rel="stylesheet" href="styles.css" />
```
Comment (nosovk) [inline index.html]: absolute links?

File: package.json [hunk 8]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "web1",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "eslint .",
```
Comment (nosovk) [inline package.json]: lint is not test

Comment (nosovk) [reply package.json]: eslint should ignore folders specified in .gitignore

File: package.json [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+    "format": "prettier --write .",
```
Comment (nosovk) [inline package.json]: there should be 2 ways of linters/formaters invocation: intrusive and not. Probably we missing something here

File: styles.css [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+  grid-area: span 1 / 1;
+}
+
+#b3 {
+  background: #f38200;
+  grid-area: span 2 / 2 / 3 / 3;
+}
+
+#b4 {
+  background: #4964ff;
```
Comment (nosovk) [inline styles.css]: css vars?

**Comment by nosovk** [comment]
github actions to launch validations?

**Comment by 4uhaister** [comment]
fixed

**Pull request closed without merging**